types/office-js-preview/index.d.ts

```diff
@@ -15569,24 +15569,95 @@ declare namespace Office {
          */
         addItemAttachmentAsync(itemId: any, attachmentName: string, callback?: (asyncResult: Office.AsyncResult<string>) => void): void;
         /**
-         * Closes the current item that is being composed
+         * Closes the current item that is being composed.
          *
-         * The behaviors of the close method depends on the current state of the item being composed.
+         * The behavior of the close method depends on the current state of the item being composed.
          * If the item has unsaved changes, the client prompts the user to save, discard, or close the action.
          *
          * In the Outlook desktop client, if the message is an inline reply, the close method has no effect.
          *
-         * **Note**: In Outlook on the web, if the item is an appointment and it has previously been saved using `saveAsync`, the user is prompted to save,
-         * discard, or cancel even if no changes have occurred since the item was last saved.
-         *
          * @remarks
          * [Api set: Mailbox 1.3]
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **restricted**
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Compose
+         * 
+         * **Important**: In Outlook on the web, if the item is an appointment and it has previously been saved using `saveAsync`, the user is prompted to save,
+         * discard, or cancel even if no changes have occurred since the item was last saved.
+         * 
+         * **Tip**: Use the
+         * {@link https://learn.microsoft.com/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-closeasync-member(1) | closeAsync}
+         * method instead of the `close` method if you want your add-in to:
+         * 
+         * - Automatically discard a message being composed without prompting the user with the save dialog.
+         * 
+         * - Determine when a user cancels the save item dialog on a message being composed.
+         * 
+         * - Close an inline reply or existing draft from an Outlook desktop client.
```

Comment:
An "inline" reply is just a reply, isn't it? It's not some special kind of reply. If that's right, then let's remove the word "inline". It confused me and I think will confuse readers.

Reply:
So what does "inline reply" mean? "Inline" usually means writing additional lines between lines of the original message. But Outlook doesn't know if you are doing that or writing a complete reply message, so that can't be what "inline" means.

Reply:
perhaps instead of "inline reply", a better term would be something like "reply in the Reading Pane" if this isn't just a reply and there's some distinction from a reply in its own window.

Reply:
Thank you both for the feedback. The method applies to all replies, including those popped out in a new window, so I've changed it to just "reply". In the Remarks section, the distinction is necessary, so I changed it "reply in the Reading Pane" to make it clearer.